transformers/storage/flap/repository_test.go

```diff
@@ -319,12 +319,22 @@ var _ = Describe("Flap storage repository", func() {
 				IsAMapping:     true,
 				Schema:         constants.MakerSchema,
 				TableName:      constants.FlapBidBidTable,
-				Repository:     &repository,
+				Repository:     repository,
 				Metadata:       bidBidMetadata,
 			}
-
 			shared_behaviors.SharedStorageRepositoryBehaviors(&inputs)
 
+			triggerInput := shared_behaviors.BidTriggerTestInput{
+				Repository:      repository,
+				Metadata:        bidBidMetadata,
+				ContractAddress: repository.ContractAddress,
+				TriggerTable:    constants.FlapTable,
+				FieldTable:      constants.FlapBidBidTable,
+				ColumnName:      constants.BidColumn,
+			}
+			shared_behaviors.CommonBidSnapshotTriggerTests(triggerInput)
+			shared_behaviors.SharedBidHistoryTriggerTests(triggerInput)
```

Comment:
Yeah although it isn't very clear. Since there are 2 separate bid structures at play, i.e. the one used by flap + flop vs the one used by flip (where the flap/flop one is a subset of the flip one), I'm trying to use the dichotomy between a "common bid" vs a "flip bid" to express that distinction. Hence why we're using `CommonBidSnapshot...` in this suite and `FlipBidSnapshot...` in the flip storage suite.

OTOH, by `SharedBidHistoryTriggerTests`, I just mean "tests about behavior that's shared across all bid history triggers".

So to recap, `Common` is being used as a way to distinguish between types of bids, and `Shared` is being used to describe what the tests are about. I'll try to think of a less confusing way to imply those ideas haha. Open to suggestions!

Reply:
I wonder if removing the `Common` and `Shared` would be okay? So just have `BidSnapshotTriggerTests` and `BidHistoryTriggerTests` and we can infer that they're common or shared by the fact that they're used in multiple places. And then `FlipBidSnapshot...` would stay named that way to explicitly point out that Flips are different?

Reply:
Great idea! Will update. FYI I've been enacting feedback from this PR onto the reorg PR, just because a lot of the suggested changes are also applicable to that one, and it's easier to make them all at once.